notice/store.go

```diff
@@ -78,3 +83,58 @@ func (s *Store) FindAllPolicyNotices(ctx context.Context, policyID string) ([]No
 
 	return notices, nil
 }
+
+// FindAllServiceNotices returns any relevant notices for the given service. Currently returns
+// notices pertaining to the system limit for unacknowledged alerts.
+func (s *Store) FindAllServiceNotices(ctx context.Context, serviceID string) ([]Notice, error) {
+	err := permission.LimitCheckAny(ctx, permission.User)
+	if err != nil {
+		return nil, err
+	}
+
+	err = validate.UUID("serviceID", serviceID)
+	if err != nil {
+		return nil, err
+	}
+
+	count, err := gadb.New(s.db).CountUnackedAlertsByService(ctx, uuid.MustParse(serviceID))
+	if err != nil {
+		return nil, err
+	}
+
+	ls, err := limit.NewStore(ctx, s.db)
+	if err != nil {
+		return nil, err
+	}
+	l, err := ls.All(ctx)
+	if err != nil {
+		return nil, err
+	}
+	unackedLim := int64(limit.Limits.Max(l, "unacked_alerts_per_service"))
+	details := fmt.Sprintf("New alerts will be rejected while at or above the limit (%v), acknowledge or close alerts to resolve.", count)
+
+	// nearing system limit notice
+	p := float32(count) / float32(unackedLim)
+	if p > 0.75 && count < unackedLim {
+		return []Notice{
+			{
+				Type:    TypeWarning,
+				Message: "Near unacknowledged alert limit",
+				Details: details,
+			},
+		}, nil
+	}
+
+	// hit system limit notice
+	if count >= unackedLim {
+		return []Notice{
+			{
+				Type:    TypeError,
+				Message: "Unacknowledged alert limit reached",
+				Details: details,
+			},
+		}, nil
+	}
```

Comment:
nit: do the error case first, and then drop the `count < unackedLim` from the warning check.